.github/workflows/snap.yaml

```diff
@@ -16,11 +16,16 @@ on:
 
 jobs:
   snap:
-    uses: ubuntu-robotics/snap_workflows/.github/workflows/snap.yaml@main
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: ['["ubuntu-latest"]', '["ARM64", "self-hosted", "linux", "large", "noble"]']
```

Comment:
Wouldn't `image` be more appropriate since this is assigned to the `ubuntu-image` var?

Reply:
it's not only an image. Maybe `machine` would be more appropriate?

Reply:
How about simply `runner` ?

Reply:
works for me I will update.